index.d.ts

```diff
@@ -4,10 +4,12 @@ import { ConnectionOptions } from 'pg-connection-string';
 import { RedisClientOptions } from 'redis';
 import { RedisOptions } from 'ioredis';
 import { MongoClientOptions } from 'mongodb';
+import { Options } from 'amqplib';
```

Comment:
This does not look right to me.  It should be the options for rhea. amqplib - https://www.npmjs.com/package/amqplib does not look like a match for that. 

The typings for rhea seem to be provided in the npm install under:

/home/midawson/checkit2/node_modules/rhea/typings

We'd have to check by building a simple typescript app that used kube-service-bindings but I think it my be

import { ConnectionOptions as rheaConnectionOptions} from 'rhea';
export function getBinding(service: 'AMQP', client: 'rhea'): rheaConnectionOptions;

@pacostas can you try that out with a simple typscript client?



I think the right one might be:

import {connection} from 'rhea'

Reply:
Correct, amqplib is for rabbitMQ using amq 0-9-1 and not for amqp1.0. I'll try to test it.